huaweicloud/services/identitycenter/resource_huaweicloud_identitycenter_custom_policy_attachment.go

```diff
@@ -0,0 +1,242 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product IdentityCenter
+// ---------------------------------------------------------------
+
+package identitycenter
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceCustomPolicyAttachment() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceCustomPolicyAttachmentCreate,
+		UpdateContext: resourceCustomPolicyAttachmentUpdate,
+		ReadContext:   resourceCustomPolicyAttachmentRead,
+		DeleteContext: resourceCustomPolicyAttachmentDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceCustomPolicyAttachmentImport,
+		},
+
+		Description: "schema: Internal",
+		Schema: map[string]*schema.Schema{
+			"instance_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the ID of the IAM Identity Center instance.`,
+			},
+			"permission_set_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the ID of the IAM Identity Center permission set.`,
+			},
+			"custom_policy": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the custom policy to attach to a permission set.`,
+			},
+		},
+	}
+}
+
+func resourceCustomPolicyAttachmentCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
you can merge this func with resourceCustomPolicyAttachmentUpdate

Reply:
done